packages/core/webpack.config.dist.js

```diff
@@ -14,6 +15,9 @@ module.exports = {
     libraryTarget: "umd"
   },
   plugins: [
+    new MonacoWebpackPlugin({
+      languages: ['json']
+    }),
```

Comment:
I don't think this plugin is usable within a library. I'm almost certain that we're just incurring a build time cost with no usable change in output.

Reply:
Yeah, that's right. Perhaps we could split it so it's only available for the playground and docs in a separate PR though?